infra/base-images/base-runner/run_fuzzer

```diff
@@ -26,7 +26,7 @@ DEBUGGER=${DEBUGGER:-}
 FUZZER=$1
 shift
 
-CORPUS_DIR="/tmp/${FUZZER}_corpus"
+CORPUS_DIR=${$CORPUS_DIR:-"/tmp/${FUZZER}_corpus"}
```

Comment:
There is a extra $
needs to be
${CORPUS_DIR:-

Reply:
Ah thanks